engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-repository-sets.html

```diff
@@ -16,6 +16,18 @@
            ng-model="repositorySetFilter"/>
   </div>
 
+  <div data-block="filters">
+    <label class="checkbox-inline" title="{{ 'Show all Repository Sets available to this Activation Key' | translate }}">
+      <input type="checkbox" ng-model="contentAccessModes.contentAccessModeAll" ng-change="toggleFilters()"/>
+      <span translate>Show All</span>
```

Comment:
You know, I didn't even notice the tooltip.  When you say "available" what does that mean?  I think that is my confusion.